scss/bootstrap-grid.scss

```diff
@@ -9,6 +9,7 @@ $include-column-box-sizing: true !default;
 
 @import "mixins/lists";
 @import "mixins/breakpoints";
+@import "mixins/color-mode";
```

Comment:
IMO we could improve the architecture around `_root.scss`; we shouldn't have to embed the colors (from `_root.scss` and `mixins/color-mode`) in `bootstrap-grid.scss`.
TBH I haven't checked in details but I suppose we only need the `@each` loop with `$grid-breakpoints` that could be extracted in a separate file and so we could only import this file here instead of `mixins/color-mode` and `root`. This `bootstrap-grid.scss` would be lighter.